source/pins.cmake

```diff
@@ -0,0 +1,18 @@
+set(pinScript  "${CMAKE_CURRENT_LIST_DIR}/pins.py")
+set(pinFileDir "${CMAKE_CURRENT_LIST_DIR}/../generated")
```

Comment:
Initially, there were some problems with this, but I'll see if I can resolve them.